scripts/skills/perks/perk_rf_dismemberment.nut

```diff
@@ -0,0 +1,71 @@
+this.perk_rf_dismemberment <- ::inherit("scripts/skills/skill", {
+	m = {
+		IsForceEnabled = false,
```

Comment:
A comment here would be nice to summarize what exactly IsForceEnabled does for *this* perk.

Like `// ignore axe weapon requirement` or just `// ignore weapon requirement`